deploy/openshift.template.yaml

```diff
@@ -33,15 +32,18 @@ objects:
           app: git-partition-sync-producer
       spec:
         serviceAccountName: git-partition-sync-producer
-        volumes:
```

Comment:
What's the story here?

Reply:
gpg key is not being utilized. Opting for https://github.com/FiloSottile/age x25519 keys. Those can be set as environment variables instead of the hassle of working with opengpg to read local key file.